docs/shared/versioning.md

```diff
@@ -0,0 +1,25 @@
+# Android Versioning
+
+This repository uses the same `version.txt` to define the version for Android Components, Focus, and Fenix. The following formatting corresonds to release branches 
+on the release track.
+
+## Format
+
+XXX.0 (where XXX is defined as the current desktop release #)
+
+*Please note: the initial release is `major.minor`, but the following releases will be `major.minor.patch`*
+
+* Levaraging semantic versioning, the first and second digit will be directly tied to the associated desktop release 
+* The third digit will be reserved for Android-only patch releases
+
+### Example
+|                                | Desktop               | Android                        |
+| ------------------------------ | --------------------- | ------------------------------ |
+| Fx XXX initial release         | XXX.0                 | XXX.0                          |
+| Desktop + Android dot release  | XXX.0.1               | XXX.1.0                        |
+| Android-only patch             | <sub>No change</sub>  | XXX.1.1                        |
```

Comment:
Not sure if it matters in practice, but the other "No change" entries all have a space after them before the closing </sub> tag.